components/support/support-additional.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "id": 20000,
+    "name": "MoonMail",
+    "username": "moonmail",
+    "tier": "sponsor",
+    "avatar": "https://static.moonmail.io/moonmail-logo.svg",
```

Comment:
Idk, I kind of like keeping these external for now. I think our asset management still has some room for improvement so I'd like to avoid adding any more than there already are for now. Plus, this stays consistent with the rest of the supporter images which are also fetched externally from the opencollective urls.